app/models/portal/offering.rb

```diff
@@ -52,6 +58,10 @@ def saveables
     multiple_choices + open_responses
   end
 
+  def resource_page?
+    self.runnable.is_a? ResourcePage
+  end
+
```

Comment:
This looks like it is left over from the old code. ResourcePages have been removed from the portal.